app/routes/socialEvents.tsx

```diff
@@ -0,0 +1,15 @@
+import EventCard from '~/components/common/eventCard';
+
+export default function socialEvents() {
+  return (
+    <>
+      <EventCard
+        date={30}
```

Comment:
Instead of hard coding, can we maintain constants? And if needed create helper files for formatting, eg:
We may not get a date as 30 or a month as AUG. 
Do we need some sort of formating here? Check with backend how they are going to send data

Reply:
Did we thing of this?